pq/stateful_hash_sig/0001-Patch-to-support-wolfSSL-xmss-reference-integration.patch

```diff
@@ -0,0 +1,2432 @@
+From 12dd009393d254202d0dc26763c95e2a731d5e7e Mon Sep 17 00:00:00 2001
```

Comment:
After looking through the whole patch I think some documentation needs to be added here.  Haven't looked in the README yet, but if its there, it can be pasted here.

Reply:
Agree on both points. I'll add more comments explaining the patch in an updated README for the xmss-reference patch, and some more comments.

The TLDR is these changes were needed to make xmss-reference compatible with a wolfboot build for an embedded target.

Reply:
The patch now includes an addendum readme, `patch_readme.md`, that lists all changes made, and explains their rationale. It also points to comments added to xmss-reference code giving more details.